pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "foursight-core"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
I'd minor version this, as it's a substantial change to the UI and other info reported.